extensions/machine-learning/src/test/views/models/registerModelWizard.test.ts

```diff
@@ -7,24 +7,78 @@ import * as azdata from 'azdata';
 import * as should from 'should';
 import 'mocha';
 import { createContext } from './utils';
-import { ListModelsEventName, ListAccountsEventName, ListSubscriptionsEventName, ListGroupsEventName, ListWorkspacesEventName, ListAzureModelsEventName, ModelSourceType, ListDatabaseNamesEventName, ListTableNamesEventName } from '../../../views/models/modelViewBase';
+import { ListModelsEventName, ListAccountsEventName, ListSubscriptionsEventName, ListGroupsEventName, ListWorkspacesEventName, ListAzureModelsEventName, ModelSourceType, ListDatabaseNamesEventName, ListTableNamesEventName, VerifyImportTableEventName } from '../../../views/models/modelViewBase';
 import { ImportedModel } from '../../../modelManagement/interfaces';
 import { azureResource } from '../../../typings/azure-resource';
 import { Workspace } from '@azure/arm-machinelearningservices/esm/models';
 import { ViewBase } from '../../../views/viewBase';
 import { WorkspaceModel } from '../../../modelManagement/interfaces';
 import { ImportModelWizard } from '../../../views/models/manageModels/importModelWizard';
+import { DatabaseTable } from '../../../prediction/interfaces';
 
+let accounts: azdata.Account[] = [
```

Comment:
nothing to do with this R but this reminds me that when I tried the AML flow it was not able to remember anything about azure account - can that be made better? let me know if you have something on that or should i open a followup task